src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -1562,7 +1418,7 @@ protected void resolveOnce() { }
 	 * removed from it (if supported). There are other mechanisms for that e.g.
 	 * {@link #doRefreshChildren()} (see {@link Feed} for an example).
 	 */
-	public synchronized final void syncResolve() {
+	public synchronized void syncResolve() {
```

Comment:
@SubJunk because I made deprecated method `resolve()` as private and changed it to `syncResolve()` in each asociated clases. The final cannot be overriden.